### PR TITLE
Apple HomePage added

### PR DESCRIPTION
**Pull request by saadanzarii**
Apple HOmepage only looks good on big screen

### CODE REVIEW

File: 18 - Apple HomePage/css/style.css [hunk 1]
```diff
@@ -0,0 +1,382 @@
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    outline: none;
```
Comment (epicdeepak) [inline 18 - Apple HomePage/css/style.css]: move this file directly in the "apple homepage' folder and delete the folder "CSS".

File: 18 - Apple HomePage/html/index.html [hunk 2]
```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (epicdeepak) [inline 18 - Apple HomePage/html/index.html]: delete the folder "HTML" and put index file directly in "apple homepage" folder.

**Review by epicdeepak — CHANGES_REQUESTED** [review]
Please make the mentioned changes.

**Comment by saadanzarii** [comment]
> Please make the mentioned changes.

i did't all the things as u said

**Review by epicdeepak — CHANGES_REQUESTED** [review]
image
image
image

Please check with image src, images are not getting rendered properly.

**Comment by saadanzarii** [comment]
I did my best to resolve all the issue that u raised

**Comment by epicdeepak** [comment]
> I did my best to resolve all the issue that u raised

You did a great jon @Saadansariii .

**Pull request merged**